fw/hashseg.py

```diff
@@ -191,10 +191,62 @@ def pack(self):
 			+ self.signature + self.cert_chain
 
 
+@dataclass
+class HashSegmentV7(_HashSegment):
+	version: int = 7  # Header version number
+
+	common_metadata_size: int = 24 # This defaults to 24
+	metadata_size_qcom: int = 0  # Size of metadata from Qualcomm
+	metadata_size: int = 0	# Size of metadata from OEM
+	code_size: int = 0
```

Comment:
```suggestion
	hash_table_size: int = 0
```

coreboot has a different name for this field for some reason.

Reply:
Not sure what's going on here. This field is always zero when secure boot is not enabled, so it does not matter too much tbough. I've checked this field is indeed called `code_size` in the source. Will double check later.